hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestContainerStateMachineStream.java

```diff
@@ -124,7 +124,7 @@ public void setup() throws Exception {
         MiniOzoneCluster.newBuilder(conf)
             .setNumDatanodes(3)
             .setHbInterval(200)
-            .setDataStreamMinPacketSize(1024)
+            .setDataStreamMinPacketSize(1) // 1MB
```

Comment:
This test sets size unit:

https://github.com/apache/ozone/blob/dbd2de9bbbdef0441e0b959081e3ef229c0aaa04/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestContainerStateMachineStream.java#L132

so this part of the patch should be omitted:

```suggestion
            .setDataStreamMinPacketSize(1024)
```

Reply:
@adoroszlai , Oops, you are right that the unit is set a few lines below.  Reverted the change.